src/targets/gpu/mlir.cpp

```diff
@@ -498,7 +499,7 @@ struct mlir_program
             auto ops  = create_operation_state(name);
             ops.add_attribute_value(get_operator_value(ins->get_operator()));
             if(ins->name() != "@return")
-                ops.add_results({get_shape(ins)});
+                ops.add_results({get_shape(ins)}, ins->name() != "multibroadcast");
```

Comment:
We would need to this for more then just broadcast. Transpose will have a non-standard shape as well and output of many operators will propagate the permutation as well so they all will be non-standard.